src/commands/run.ts

```diff
@@ -104,6 +111,12 @@ export function addRun(program: commander.Command): void {
         console.error(err);
       }
 
+      if (type === 'pre-commit') {
+        console.log();
+        console.log(chalk.bgCyan(unstashMessage));
+        execSync('git stash pop');
```

Comment:
Same here, yellow + bold, + would be nice to not always display this if not needed

Reply:
Resolved in 0e1647dca9a5748ac6f7ce9b540144e526e878f1